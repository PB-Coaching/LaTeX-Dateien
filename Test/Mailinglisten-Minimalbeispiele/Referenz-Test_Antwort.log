This is pdfTeX, Version 3.1415926-2.5-1.40.14 (TeX Live 2013) (format=pdflatex 2014.3.29)  2 APR 2014 17:05
entering extended mode
 restricted \write18 enabled.
 %&-line parsing enabled.
**Referenz-Test_Antwort.tex
(./Referenz-Test_Antwort.tex
LaTeX2e <2011/06/27>
Babel <3.9f> and hyphenation patterns for 42 languages loaded.
(/usr/share/texmf-dist/tex/latex/koma-script/scrreprt.cls
Document Class: scrreprt 2012/07/29 v3.11b KOMA-Script document class (report)
(/usr/share/texmf-dist/tex/latex/koma-script/scrkbase.sty
Package: scrkbase 2012/07/29 v3.11b KOMA-Script package (KOMA-Script-dependent 
basics and keyval usage)

(/usr/share/texmf-dist/tex/latex/koma-script/scrbase.sty
Package: scrbase 2012/07/29 v3.11b KOMA-Script package (KOMA-Script-independent
 basics and keyval usage)

(/usr/share/texmf-dist/tex/latex/graphics/keyval.sty
Package: keyval 1999/03/16 v1.13 key=value parser (DPC)
\KV@toks@=\toks14
)
(/usr/share/texmf-dist/tex/latex/koma-script/scrlfile.sty
Package: scrlfile 2012/06/15 v3.12 KOMA-Script package (loading files)

Package scrlfile, 2012/06/15 v3.12 KOMA-Script package (loading files)
                  Copyright (C) Markus Kohm

))) (/usr/share/texmf-dist/tex/latex/koma-script/tocbasic.sty
Package: tocbasic 2012/04/04 v3.10b KOMA-Script package (handling toc-files)
)
Package tocbasic Info: omitting babel extension for `toc'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `toc' on input line 117.
Package tocbasic Info: omitting babel extension for `lof'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `lof' on input line 118.
Package tocbasic Info: omitting babel extension for `lot'
(tocbasic)             because of feature `nobabel' available
(tocbasic)             for `lot' on input line 119.
Class scrreprt Info: File `scrsize11pt.clo' used instead of
(scrreprt)           file `scrsize11.clo' to setup font sizes on input line 136
7.

(/usr/share/texmf-dist/tex/latex/koma-script/scrsize11pt.clo
File: scrsize11pt.clo 2012/07/29 v3.11b KOMA-Script font size class option (11p
t)
)
(/usr/share/texmf-dist/tex/latex/koma-script/typearea.sty
Package: typearea 2012/07/29 v3.11b KOMA-Script package (type area)

Package typearea, 2012/07/29 v3.11b KOMA-Script package (type area)
                  Copyright (C) Frank Neukam, 1992-1994
                  Copyright (C) Markus Kohm, 1994-

\ta@bcor=\skip41
\ta@div=\count79
\ta@hblk=\skip42
\ta@vblk=\skip43
\ta@temp=\skip44
Package typearea Info: These are the values describing the layout:
(typearea)             DIV  = 10
(typearea)             BCOR = 0.0pt
(typearea)             \paperwidth      = 597.50793pt
(typearea)              \textwidth      = 418.25555pt
(typearea)              DIV departure   = -6%
(typearea)              \evensidemargin = 17.3562pt
(typearea)              \oddsidemargin  = 17.3562pt
(typearea)             \paperheight     = 845.04694pt
(typearea)              \textheight     = 595.80026pt
(typearea)              \topmargin      = -25.16531pt
(typearea)              \headheight     = 17.0pt
(typearea)              \headsep        = 20.40001pt
(typearea)              \topskip        = 11.0pt
(typearea)              \footskip       = 47.60002pt
(typearea)              \baselineskip   = 13.6pt
(typearea)              on input line 1213.
)
\c@part=\count80
\c@chapter=\count81
\c@section=\count82
\c@subsection=\count83
\c@subsubsection=\count84
\c@paragraph=\count85
\c@subparagraph=\count86
\abovecaptionskip=\skip45
\belowcaptionskip=\skip46
\c@pti@nb@sid@b@x=\box26
\c@figure=\count87
\c@table=\count88
\bibindent=\dimen102
) (/usr/share/texmf-dist/tex/latex/base/fontenc.sty
Package: fontenc 2005/09/27 v1.99g Standard LaTeX package

(/usr/share/texmf-dist/tex/latex/base/t1enc.def
File: t1enc.def 2005/09/27 v1.99g Standard LaTeX file
LaTeX Font Info:    Redeclaring font encoding T1 on input line 43.
))
(/usr/share/texmf-dist/tex/latex/hyperref/hyperref.sty
Package: hyperref 2012/11/06 v6.83m Hypertext links for LaTeX

(/usr/share/texmf-dist/tex/generic/oberdiek/hobsub-hyperref.sty
Package: hobsub-hyperref 2012/05/28 v1.13 Bundle oberdiek, subset hyperref (HO)


(/usr/share/texmf-dist/tex/generic/oberdiek/hobsub-generic.sty
Package: hobsub-generic 2012/05/28 v1.13 Bundle oberdiek, subset generic (HO)
Package: hobsub 2012/05/28 v1.13 Construct package bundles (HO)
Package: infwarerr 2010/04/08 v1.3 Providing info/warning/error messages (HO)
Package: ltxcmds 2011/11/09 v1.22 LaTeX kernel commands for general use (HO)
Package: ifluatex 2010/03/01 v1.3 Provides the ifluatex switch (HO)
Package ifluatex Info: LuaTeX not detected.
Package: ifvtex 2010/03/01 v1.5 Detect VTeX and its facilities (HO)
Package ifvtex Info: VTeX not detected.
Package: intcalc 2007/09/27 v1.1 Expandable calculations with integers (HO)
Package: ifpdf 2011/01/30 v2.3 Provides the ifpdf switch (HO)
Package ifpdf Info: pdfTeX in PDF mode is detected.
Package: etexcmds 2011/02/16 v1.5 Avoid name clashes with e-TeX commands (HO)
Package etexcmds Info: Could not find \expanded.
(etexcmds)             That can mean that you are not using pdfTeX 1.50 or
(etexcmds)             that some package has redefined \expanded.
(etexcmds)             In the latter case, load this package earlier.
Package: kvsetkeys 2012/04/25 v1.16 Key value parser (HO)
Package: kvdefinekeys 2011/04/07 v1.3 Define keys (HO)
Package: pdftexcmds 2011/11/29 v0.20 Utility functions of pdfTeX for LuaTeX (HO
)
Package pdftexcmds Info: LuaTeX not detected.
Package pdftexcmds Info: \pdf@primitive is available.
Package pdftexcmds Info: \pdf@ifprimitive is available.
Package pdftexcmds Info: \pdfdraftmode found.
Package: pdfescape 2011/11/25 v1.13 Implements pdfTeX's escape features (HO)
Package: bigintcalc 2012/04/08 v1.3 Expandable calculations on big integers (HO
)
Package: bitset 2011/01/30 v1.1 Handle bit-vector datatype (HO)
Package: uniquecounter 2011/01/30 v1.2 Provide unlimited unique counter (HO)
)
Package hobsub Info: Skipping package `hobsub' (already loaded).
Package: letltxmacro 2010/09/02 v1.4 Let assignment for LaTeX macros (HO)
Package: hopatch 2012/05/28 v1.2 Wrapper for package hooks (HO)
Package: xcolor-patch 2011/01/30 xcolor patch
Package: atveryend 2011/06/30 v1.8 Hooks at the very end of document (HO)
Package atveryend Info: \enddocument detected (standard20110627).
Package: atbegshi 2011/10/05 v1.16 At begin shipout hook (HO)
Package: refcount 2011/10/16 v3.4 Data extraction from label references (HO)
Package: hycolor 2011/01/30 v1.7 Color options for hyperref/bookmark (HO)
)
(/usr/share/texmf-dist/tex/generic/ifxetex/ifxetex.sty
Package: ifxetex 2010/09/12 v0.6 Provides ifxetex conditional
)
(/usr/share/texmf-dist/tex/latex/oberdiek/auxhook.sty
Package: auxhook 2011/03/04 v1.3 Hooks for auxiliary files (HO)
)
(/usr/share/texmf-dist/tex/latex/oberdiek/kvoptions.sty
Package: kvoptions 2011/06/30 v3.11 Key value format for package options (HO)
)
\@linkdim=\dimen103
\Hy@linkcounter=\count89
\Hy@pagecounter=\count90

(/usr/share/texmf-dist/tex/latex/hyperref/pd1enc.def
File: pd1enc.def 2012/11/06 v6.83m Hyperref: PDFDocEncoding definition (HO)
)
\Hy@SavedSpaceFactor=\count91

(/usr/share/texmf-dist/tex/latex/latexconfig/hyperref.cfg
File: hyperref.cfg 2002/06/06 v1.2 hyperref configuration of TeXLive
)
Package hyperref Info: Option `bookmarks' set `true' on input line 4319.
Package hyperref Info: Option `pdftoolbar' set `true' on input line 4319.
Package hyperref Info: Option `pdfmenubar' set `true' on input line 4319.
Package hyperref Info: Option `breaklinks' set `true' on input line 4319.
Package hyperref Info: Hyper figures OFF on input line 4443.
Package hyperref Info: Link nesting OFF on input line 4448.
Package hyperref Info: Hyper index ON on input line 4451.
Package hyperref Info: Plain pages OFF on input line 4458.
Package hyperref Info: Backreferencing OFF on input line 4463.
Package hyperref Info: Implicit mode ON; LaTeX internals redefined.
Package hyperref Info: Bookmarks ON on input line 4688.
\c@Hy@tempcnt=\count92

(/usr/share/texmf-dist/tex/latex/url/url.sty
\Urlmuskip=\muskip10
Package: url 2006/04/12  ver 3.3  Verb mode for urls, etc.
)
LaTeX Info: Redefining \url on input line 5041.
\XeTeXLinkMargin=\dimen104
\Fld@menulength=\count93
\Field@Width=\dimen105
\Fld@charsize=\dimen106
Package hyperref Info: Hyper figures OFF on input line 6295.
Package hyperref Info: Link nesting OFF on input line 6300.
Package hyperref Info: Hyper index ON on input line 6303.
Package hyperref Info: backreferencing OFF on input line 6310.
Package hyperref Info: Link coloring OFF on input line 6315.
Package hyperref Info: Link coloring with OCG OFF on input line 6320.
Package hyperref Info: PDF/A mode OFF on input line 6325.
LaTeX Info: Redefining \ref on input line 6365.
LaTeX Info: Redefining \pageref on input line 6369.
\Hy@abspage=\count94
\c@Item=\count95
\c@Hfootnote=\count96
)

Package hyperref Message: Driver (autodetected): hpdftex.

(/usr/share/texmf-dist/tex/latex/hyperref/hpdftex.def
File: hpdftex.def 2012/11/06 v6.83m Hyperref driver for pdfTeX
\Fld@listcount=\count97
\c@bookmark@seq@number=\count98

(/usr/share/texmf-dist/tex/latex/oberdiek/rerunfilecheck.sty
Package: rerunfilecheck 2011/04/15 v1.7 Rerun checks for auxiliary files (HO)
Package uniquecounter Info: New unique counter `rerunfilecheck' on input line 2
82.
)
\Hy@SectionHShift=\skip47
)
(./Referenz-Test_Antwort.aux)
\openout1 = `Referenz-Test_Antwort.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
LaTeX Font Info:    Checking defaults for PD1/pdf/m/n on input line 13.
LaTeX Font Info:    ... okay on input line 13.
\AtBeginShipoutBox=\box27
Package hyperref Info: Link coloring OFF on input line 13.

(/usr/share/texmf-dist/tex/latex/hyperref/nameref.sty
Package: nameref 2012/10/27 v2.43 Cross-referencing by name of section

(/usr/share/texmf-dist/tex/generic/oberdiek/gettitlestring.sty
Package: gettitlestring 2010/12/03 v1.4 Cleanup title references (HO)
)
\c@section@level=\count99
)
LaTeX Info: Redefining \ref on input line 13.
LaTeX Info: Redefining \pageref on input line 13.
LaTeX Info: Redefining \nameref on input line 13.

(./Referenz-Test_Antwort.out) (./Referenz-Test_Antwort.out)
\@outlinefile=\write3
\openout3 = `Referenz-Test_Antwort.out'.

LaTeX Font Info:    Try loading font information for T1+cmss on input line 16.

(/usr/share/texmf-dist/tex/latex/base/t1cmss.fd
File: t1cmss.fd 1999/05/25 v2.5h Standard LaTeX font definitions
)
Package atveryend Info: Empty hook `BeforeClearDocument' on input line 20.
 [1

{/var/lib/texmf/fonts/map/pdftex/updmap/pdftex.map}]
Package atveryend Info: Empty hook `AfterLastShipout' on input line 20.
 (./Referenz-Test_Antwort.aux)
Package atveryend Info: Executing hook `AtVeryEndDocument' on input line 20.
Package atveryend Info: Executing hook `AtEndAfterFileList' on input line 20.
Package rerunfilecheck Info: File `Referenz-Test_Antwort.out' has not changed.
(rerunfilecheck)             Checksum: DACD9587B05B60AD3F798FC6A6067892;84.
Package atveryend Info: Empty hook `AtVeryVeryEnd' on input line 20.
 ) 
Here is how much of TeX's memory you used:
 5583 strings out of 493853
 85184 string characters out of 6157277
 187841 words of memory out of 5000000
 8863 multiletter control sequences out of 15000+600000
 7029 words of font info for 19 fonts, out of 8000000 for 9000
 788 hyphenation exceptions out of 8191
 36i,6n,35p,235b,319s stack positions out of 5000i,500n,10000p,200000b,80000s
{/usr/share/texmf-dist/fonts/enc/dvips/cm-super/cm-super-t1.enc}</usr/share/t
exmf-dist/fonts/type1/public/cm-super/sfrm1095.pfb></usr/share/texmf-dist/fonts
/type1/public/cm-super/sfss1095.pfb></usr/share/texmf-dist/fonts/type1/public/c
m-super/sfsx1440.pfb>
Output written on Referenz-Test_Antwort.pdf (1 page, 40752 bytes).
PDF statistics:
 32 PDF objects out of 1000 (max. 8388607)
 25 compressed objects within 1 object stream
 3 named destinations out of 1000 (max. 500000)
 9 words of extra memory for PDF output out of 10000 (max. 10000000)

