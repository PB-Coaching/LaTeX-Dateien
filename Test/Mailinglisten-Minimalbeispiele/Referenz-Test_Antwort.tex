\documentclass{scrreprt}

\usepackage[T1]{fontenc}
\usepackage[bookmarks=true,pdftoolbar=true,
pdfmenubar=true,pdfstartview={FitH},
pdftitle={Gasmarktliberalisierung im Baltikum},
pdfauthor={Pascal Bernhard},
pdfproducer={Texlive-LaTeX 2013 on
Sabayon GNU/Linu 3.12},citecolor=magenta,
%hidelinks,%The link is not hidden anymore
breaklinks=true]{hyperref}

\begin{document}

\section[Name of the section to be used by
the reference]{Section 1}\label{sec:Section1}

As you can see in section \textsf{\nameref{sec:Section1}}

\end{document}