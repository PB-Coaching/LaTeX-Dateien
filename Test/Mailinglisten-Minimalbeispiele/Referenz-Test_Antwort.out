\BOOKMARK [1][-]{section.0.1}{Name of the section to be used by the reference}{}% 1
